packages/orbit-components/src/Wizard/index.tsx

```diff
@@ -100,12 +99,7 @@ const Wizard = ({
           }}
         >
           <Stack as="span" inline>
-            <b>
-              {translate("wizard_progress", {
-                number: activeStep + 1,
-                total: stepsCount,
-              })}
-            </b>
+            <b>{labelProgress}</b>
```

Comment:
Stories need to be updated, as this was being applied automatically. They need to get this prop with the current step, the string and the total steps